src/widgets/barchart/bar.rs

```diff
@@ -206,3 +230,52 @@ impl<'a> Bar<'a> {
         }
     }
 }
+
+impl<'a> From<(&'a str, u64)> for Bar<'a> {
+    fn from((label, value): (&'a str, u64)) -> Self {
+        Bar::new(label, value)
+    }
+}
```

Comment:
It is a bit unconventional to use tuples in `From` implementations. In other words, we don't have much examples of that in the codebase. Not a big problem though - I think it should be documented well.

Reply:
Where should I write the document about it? Top of the `From` implementation or somewhere else? I can try my best to document it, or removing it is also an option if others decide on this direction

Reply:
Let's wait for other maintainers' feedback first :)